matchzoo/models/cdssm_model.py

```diff
@@ -33,59 +35,63 @@ class CDSSMModel(engine.BaseModel):
     @classmethod
     def get_default_params(cls) -> engine.ParamTable:
         """:return: model default parameters."""
-        params = super().get_default_params(with_multi_layer_perceptron=True)
-        params['optimizer'] = 'adam'
-        params['input_shapes'] = [(10, 900), (10, 900)]
-        params.add(engine.Param('w_initializer', 'glorot_normal'))
-        params.add(engine.Param('b_initializer', 'zeros'))
+        # set :attr:`with_multi_layer_perceptron` to False to support
+        # user-defined variable dense layer units
+        params = super().get_default_params()
         params.add(engine.Param('filters', 32))
         params.add(engine.Param('kernel_size', 3))
         params.add(engine.Param('strides', 1))
         params.add(engine.Param('padding', 'same'))
-        params.add(engine.Param('conv_activation_func', 'tanh'))
+        params.add(engine.Param('conv_activation_func', 'relu'))
+        params.add(engine.Param('w_initializer', 'glorot_normal'))
+        params.add(engine.Param('b_initializer', 'zeros'))
+        params.add(engine.Param('dropout_rate', 0.3))
+        params.add(engine.Param('mlp_activation_func', 'relu'))
+        params.add(engine.Param('mlp_hidden_units', [64, 32]))
```

Comment:
If you understand how `matchzoo.engine.hyper_spaces` works, add at least one here, so I could test auto tunning.